README.md

```diff
@@ -28,17 +28,17 @@ curl_easy_setopt(curl, CURLOPT_FOLLOWLOCATION, 1)
 
 
 # setup the callback function to recv data
-function curl_write_cb(curlbuf::Ptr{Void}, s::Csize_t, n::Csize_t, p_ctxt::Ptr{Void})
+function curl_write_cb(curlbuf::Ptr{Cvoid}, s::Csize_t, n::Csize_t, p_ctxt::Ptr{Cvoid})
     sz = s * n
     data = Array{UInt8}(sz)
```

Comment:
I think we need this here too

```suggestion
    data = Array{UInt8}(undef, sz)
```

Reply:
https://github.com/JuliaWeb/LibCURL.jl/pull/63